kodi_addon_checker/xml_schema/jarvis_contextitem.xsd

```diff
@@ -4,7 +4,12 @@
   <xs:element name="extension">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="menu" type="menuType" minOccurs="1" maxOccurs="unbounded"/>
+        <xs:element name="menu" type="menuType" minOccurs="1" maxOccurs="unbounded">
```

Comment:
context menu items were introduced in isengard so I assume you need a similar check on other schemas too

Reply:
I see, let me fix that.

Reply:
From `isengard_contextitem.xsd` it seems that the nesting is done by using multiple `extension` nodes. As the XSD only operates on a single Node, we won't be able to validate it with an XSD.

So my suggestion would be to go forward with this PR and let Isengard be as it is.